openedx/features/idea/models.py

```diff
@@ -30,27 +37,36 @@ class Meta:
 
 
 class VisualAttachment(models.Model):
+    """
+    Model contains fields related to visual attachment
+    """
+    # size_in_mb = bytes_to_mb(IDEA_IMAGE_MAX_SIZE)
+    # help_text_str = _('Accepted extensions: .jpg, .png (maximum {mb} MB)')
     video_link = models.URLField(blank=True, null=True, verbose_name=_('VIDEO LINK'))
     image = models.ImageField(
         storage=CustomS3Storage(), max_length=500, blank=True, null=True,
         upload_to=UploadToPathAndRename(path='images', name_prefix='image', add_path_prefix=True),
         validators=[FileExtensionValidator(['jpg', 'png'], )], verbose_name=_('ADD IMAGE'),
-        help_text=_('Accepted extensions: .jpg, .png (maximum {mb} MB)'.format(mb=bytes_to_mb(IDEA_IMAGE_MAX_SIZE)))
+        help_text=format_lazy(_('Accepted extensions: .jpg, .png (maximum {mb} MB)'),
+                              mb=bytes_to_mb(IDEA_IMAGE_MAX_SIZE))
```

Comment:
Please break lines, here, in lines 58-59 and elsewhere according to the following edx recommendations:

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines